eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -174,14 +175,14 @@ extends:
 
               if [ "${{ parameters.SpecBatchTypes }}" != "" ]; then
                 optional_params="$optional_params --rm ${{ parameters.SpecBatchTypes }}"
-                sdk_gen_info="$sdk_gen_info, SpecBatchTypes: ${{ parameters.SpecBatchTypes }}"
+                sdk_gen_info="$sdk_gen_info SpecBatchTypes: ${{ parameters.SpecBatchTypes }}"
```

Comment:
Is this fixing another unrelated bug?

Reply:
yes, removing extra comma. Do you want me to have separate PR?

Reply:
No, it is fine I just wanted to make sure it wasn't a mistake.